ci

```diff
@@ -128,24 +183,47 @@ run_log_commit () {
     slack_name=${slack_name##*/}
     slack_name=${slack_name%.git}
     slack_os=$(uname)
+   
     if $success; then
       slack_emoji=":white_check_mark:"
       slack_type="success 😺"
     else
       slack_emoji=":no_entry:"
       slack_type="failure 😿"
     fi
-    if [[ ${slack_branch##origin/c_} != $slack_branch ]]; then
-      slack_channel=${slack_branch##origin/c_}
-      slack_channel=${slack_channel%%_*}
-    fi
+
+    # check if branch has c_ to send to appropriate channel - else check if has individual name in branch
+	if [[ ${slack_branch##origin/c_} != $slack_branch ]]; then
+	
+	  slack_channel=${slack_branch##origin/c_}
+	  slack_channel=${slack_channel%%_*}
+
+	else  #check individual name - use temp vars just in case doesn't pass valid member
```

Comment:
Assuming your usernames are stored in a separate file, your can condense this entire else-branch and get rid of verify_everest_team_member -- here's a sketch:

```
      ...
      slack_channel="#${slack_channel%%_*}"
    elif [[ $(egrep ^$slack_branch$ $EVEREST_CI_HOME/slack-users) == $slack_branch ]]; then
      slack_channel="@$slack_branch"
    fi
```

This requires you to add:

```
EVEREST_CI_HOME=$(pwd)
```

at line 58 right before the popd

and also requires you to change `post_to_slack` to not always prepend a `#` to the channel name (it's a leftover mistake)